### ISSUE TITLE
Drag or whatever action to move current tab to a new window

### ISSUE THREAD

**Issue by pierremtb** [lirios/browser#11]

**Comment by timsueberkrueb** [comment]
@pierremtb Any idea how this could be implemented?

**Comment by pierremtb** [comment]
Maybe we could add a dragUp and dragDown listener to your tabs engine? I don't how hard it would be though. At least a right click option in a right click menu on a tab, but that would be another issue :)

**Comment by timsueberkrueb** [comment]
> Maybe we could add a dragUp and dragDown listener to your tabs engine? I don't how hard it would be though.

@pierremtb Yeah, the difficult question is about how to handle the drag outside of the application window. I don't know how/whether this can be implemented (platform independent).

> At least a right click option in a right click menu on a tab, but that would be another issue :)

+1

**Comment by pierremtb** [comment]
I think we could only create a new window with the desired tab. It would be
enough, for now at least.

**Comment by PoisonousJohn** [comment]
Well, the way Chrome is doing it, once you leave tabs area (line) with cursor, it just creates a new window

**Comment by timsueberkrueb** [comment]
Yeah, that should be good and simple enough.
I prefer how Firefox is only creating the window as soon as you drop the tab outside of the current window, though.
